### ISSUE TITLE
ModuleDataFixInterface modules

### ISSUE THREAD

**Issue by ric2016** [fisharebest/webtrees#4446]
It would be helpful to have an icon and special column for data fix modules in the control panel (Control panel > All modules).

Right now, these modules appear with all icons empty, and from their descriptions alone it isn't always clear that they are actually data fixes, as in the example below.

<img width="818" alt="datafix" src="https://user-images.githubusercontent.com/18174787/170882171-dc05f218-5fe3-47ac-b739-c2d457006eec.PNG">

**Comment by hartenthaler** [comment]
Yes, this would allow us to sort the table by this new column and to see all data fix modules together. Good idea.

**Comment by ddrury** [comment]
They are already grouped in the module section, doesn't that suffice?

![demo](https://user-images.githubusercontent.com/104515/170994498-8d7c1cf3-2423-4502-97b0-971f019c75fd.jpg)

**Comment by ric2016** [comment]
> They are already grouped in the module section, doesn't that suffice?

No. This doesn't address this point (when viewing lthe ist of all modules):

> from their descriptions alone it isn't always clear that they are actually data fixes

Also other module kinds are grouped elsewhere and may still be grouped in the list of all modules.
